examples/next-ssr-example/README.md

```diff
@@ -4,7 +4,7 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped using [`create-nex
 
 ## Instrumenting with Splunk RUM using NPM installation method
 
-Next.js uses [static generation](https://nextjs.org/docs/basic-features/pages#pre-rendering) by default to generate the output HTML for most pages. As static generation is done using the node.js runtime, it isn't compatible with RUM. Check that RUM is only imported and activated in the browser environment.
```

Comment:
The link return 404 and link checker in CI fails. I decided to remove it. This example needs to remove anyway.